tracer/test/snapshots/RaspIast.AspNetMvc5.Classic.Lfi_url=-Iast-GetFileContent-file=filename_exploit=Lfi.verified.txt

```diff
@@ -77,7 +77,7 @@
       _dd.appsec.enabled: 1.0,
       _dd.top_level: 1.0,
       _dd.tracer_kr: 1.0,
-      _sampling_priority_v1: 2.0
+      _sampling_priority_v1: 1.0
```

Comment:
These seem surprising? 🤔

Reply:
I asked in the channel. This is the only call that actually does not involve adding ASM tags to the span. I am not used to the logic applied to this tag but, from what I understood, the ASM operations can affect this tag. 🤔